target/bin/fail2ban

```diff
@@ -72,6 +73,13 @@ else
       cat /var/log/mail/fail2ban.log
       ;;
 
+    ( 'status' )
+      for JAIL in "${JAILS[@]}"; do
+        RESULT=$(fail2ban-client status "${JAIL}" 2>&1)
+        echo -e "$RESULT\n"
```

Comment:
There is no need to store the result in a variable. This should be enough:

```suggestion
        printf '%s\n\n' "$(fail2ban-client status "${JAIL}" 2>&1)"
```

PS: I've chosen `printf` over `echo` in that case, because the input is not exactly known before and could be falsely misinterpreted as option argument to `echo` - just to be safe.